### ISSUE TITLE
Prototyping

### ISSUE THREAD

**Issue by nicholaslamon** [nicholaslamon/CNCS_Capstone#15]

**Comment by nicholaslamon** [comment]
Options for NAS

Use a RaspPi for a NAS
- Borrow from school

SD Card Storage on Dream Router

**Comment by KyleWebster46** [comment]
To configure the raspberry pi NAS, we decided to go with Open Media Vault (OMV) to use as our NAS software. OMV was installed after the pi itself was configured to run on debian, using a headless version. The Pi was then assigned an IP address, and ssh was used to actually begin configuring the pi. OMV was then installed through getting the installation script from github. After OMV was successfully installed, I began the configuration by formatting the drive attached to the pi so it could be used for storage. A base user was set up with a publicly available file share (on my private home network). I ran into a small issue with OMV, where I couldn't sign in through OMV itself. I fixed this issue by going into the OMV user settings, and re-assigning a password to the "pi" user. 

We ultimately decided we wanted the NAS to serve a greater purpose than to just hold files, so plex media server was also installed on top of OMV. This way, the NAS could play-back video footage from the security cameras. Without plex, the end user would need to install each video file before they could play it back. Plex just makes it easier by being able to play anything back instantly from the NAS itself. Plex also comes with a mobile app, which means the end user could watch the footage from their phone anywhere in the world.
